### DISCUSSION TITLE
Workload Manager Deployment Options?

### DISCUSSION THREAD

**Discussion by mocsharp** [Project-MONAI/monai-deploy-workflow-manager#29] [Q&A]
Preference for deploying the Workload Manager Core Service & Orchestration Mediator Service, and possibly the Informatics Gateway.  Please vote

♥ Kubernetes (via Helm charts)
👀 Docker-Compose

**Comment by JHancox** [comment]
I recognise that K8s + Helm is probably the 'correct' solution, but also feel that making a non K8s version available could be a real bonus (especially for testing and smaller scale installations).

**Comment by dbericat** [comment]
Please refer back to Operating Environments guideline, and define orchestration capabilities required. 

What envs are we expecting Workflow Manager to run on?
